config/sources/rk3399.conf

```diff
@@ -18,6 +18,13 @@ BOOTDELAY=0
 OVERLAY_PREFIX='rockchip'
 SERIALCON='ttyFIQ0:1500000'
 
+ATFSOURCE='https://github.com/ARM-software/arm-trusted-firmware'
```

Comment:
Please make this and above `UBOOT_TARGET_MAP` conditional for `roc-rk3399-pc` similarly to what is done in [`rockchip64-dev` family](https://github.com/armbian/build/blob/5e01f1234ed8d62569fe3ba9047780ced2d3ac79/config/sources/rockchip64.conf#L19)